doc/reference/plutus/plutus-minting-script-example.md

```diff
@@ -0,0 +1,83 @@
+# Plutus Minting Scripts
+
+## What is a Plutus minting script?
+
+A Plutus minting script is required to validate the minting of multi-asset tokens. Unlike Mary-era scripts, Plutus scripts support more logic beyond required signatures and timelocks.
+
+### An example of using a Plutus minting script
+
+Below is an example that shows how to use a Plutus minting script. This is a step-by-step
+process involving:
+
++ the creation of the `AlwaysSucceeds` Plutus minting script (i.e. anybody can mint)
++ the creation of a transaction that mints multi-assets using the `AlwaysSucceeds` Plutus minting script
+
+In this example we will use the [anyone can mint](../../../plutus-example/plutus-example/src/Cardano/PlutusExample/MintingScript.hs) Plutus minting script. To execute it, we require the following:
+
+- Collateral tx input(s) that are provided and forfeited if the Plutus script fails to execute.
+- Serialization of the Plutus script in the text envelope format (required for cardano-cli).
+- A redeemer.
+
+#### Creating the `AlwaysSucceeds` Plutus minting script
+
+The plutus-example executable will automagically generate several Plutus scripts in the CLI-compatible text envelope format.
+
+Run the following commands:
+
+```bash
+cd plutus-example
+
+cabal run exe:plutus-example
+```
+
+This will output `anyone-can-mint.plutus` in the `generated-plutus-scripts` dir.
+
+#### Setting up a local Alonzo node cluster
+
+This convenient script will set up an Alonzo cluster immediately on your local machine:
+
+```bash
+cabal install cardano-cli
+cabal install cardano-node
+./scripts/byron-to-alonzo/mkfiles.sh alonzo
+```
+
+Follow the instructions displayed in the terminal to start your Alonzo cluster.
+
+#### Minting multi-assets using the Plutus minting script
+
+To mint a multi-asset, you need the Plutus script policy ID and a redeemer. Because this minting script always succeeds, you can use any redeemer:
+
+```bash
+> cardano-cli transaction policyid --script-file scripts/plutus/scripts/anyone-can-mint.plutus
+> $policyid
+```
+
+You can find an example redeemer at: `scripts/plutus/data/42.redeemer`
+
+For more information regarding `tx-in-collateral` and `mint-execution-units` see [here](plutus-spending-script-example.md).
```

Comment:
As far as I can see `plutus-spending-script-example.md` contains no meaningful information about either those. It does use `tx-in-execution-units` and explains
> `$plutusrequiredtime` and `$plutusrequiredspace` - These make up the Plutus script execution budget and are part of the `$txfee`

but gives no information how to calculate them.

Reply:
As far as I know there is not nice way to calculate these currently. This could potentially be a new issue, however I can update the docs to use `build` instead of `build-raw` as `build` automatically calculates these for you.